tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -430,6 +430,11 @@ public static partial class Tags
         /// </summary>
         internal const string AwsQueueUrl = "aws.queue.url";
 
+        /// <summary>
+        /// The rule name associated with the AWS EventBridge span.
+        /// </summary>
+        internal const string RuleName = "rulename";
```

Comment:
I wonder if we should add the new tags right away here, like, we are aiming for it in the future, but maybe it'll be good to add it right away, either in this PR on a subsequent to not miss it. So, have a duplicate of them in this case one with `rulename` the other with `busname` whatever you think is good, and whichever right naming there is

Reply:
I talked to Harv and this is low priority, so we're gonna rename it across all runtimes in the future.

By duplicate, do you mean we should set two tags to the busname?
```
tags.RuleName = busName;
tags.BusName = busName;
```